ql/experimental/shortrate/generalizedhullwhite.hpp

```diff
@@ -217,7 +314,92 @@ namespace QuantLib {
     inline boost::shared_ptr<OneFactorModel::ShortRateDynamics>
     GeneralizedHullWhite::HWdynamics() const {
         return boost::shared_ptr<ShortRateDynamics>(
-                                            new Dynamics(phi_, a(), sigma()));
+          new Dynamics(phi_, a(), sigma()));
+    }
+
+    // todo: move to interpolations
```

Comment:
I'd keep it here for the time being.  I'd like to try and figure out if we can make some kind of extrapolation traits and cross them with the existing interpolations.